src/repositories/Repositories.ts

```diff
@@ -17,7 +17,7 @@
 import { Sequelize, InitOptions } from 'sequelize';
```

Comment:
Putting it here: please, bring back commented out lines in notify function with regards to all repos.
Also, add ContractEntityRepository to notify() function.

Reply:
Changes Done!